zuri_net/src/client/data.rs

```diff
@@ -154,6 +156,7 @@ impl Default for ClientData {
             third_party_name_only: false,
             ui_profile: 0,
             trusted_skin: false,
+            override_skin: false,
```

Comment:
I feel like trusted_skin and override_skin should both default  to true as per the comment above